Core/Executor/ContentTypeManager.php

```diff
@@ -64,11 +68,17 @@ protected function create($step)
         $contentTypeCreateStruct->nameSchema = $step->dsl['name_pattern'];
 
         // set names for the content type
-        $contentTypeCreateStruct->names = array(
-            $this->getLanguageCode($step) => $step->dsl['name'],
-        );
+        if (isset($step->dsl['names'])) {
+            $contentTypeCreateStruct->names = $step->dsl['names'];
+        } else {
```

Comment:
you forgot to add this new if else for the update function (same for description) ;)

Reply:
I added it since 5 minutes :)

Reply:
ah great! I was just looking at the code again and wondered how I overlooked this earlier. 😁